src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1663,12 +1663,29 @@ public String getContig() {
     }
 
     /**
-     * @return 1-based inclusive start position of the Variant
-     * INDEL events usually start on the first unaltered reference base before the INDEL
-     * 
-     * <strong>Warning:</strong> be aware that the start position of the VariantContext is defined in terms of the start position specified in the
-     * underlying vcf file, VariantContexts representing the same biological event may have different start positions depending on the
-     * specifics of the vcf file they are derived from
+     * Returns 1-based inclusive start position of the variant.
+     *
+     * <p>
+     *     INDEL events usually start on the first unaltered reference base before the INDEL.
+     * </p>
```

Comment:
`</p>` markup is unnecessary as the `<p>` tag closes the previous open paragraph tag.